packages/tasks/src/model-libraries.ts

```diff
@@ -662,6 +662,14 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		snippets: snippets.perception_encoder,
 		countDownloads: `path_extension:"pt"`,
 	},
+	phantom: {
```

Comment:
Unfortunately `phantom` is already used by some repos (https://huggingface.co/models?other=phantom) depending on https://github.com/ByungKwanLee/Phantom which is a totally different project.

Would it be possible to define it as `phantom-video` for instance? In any case, it will require to update the model card metadata in https://huggingface.co/bytedance-research/Phantom before getting this PR merged (with `library_name: phantom-video`)

Reply:
or `phantom-wan` to match the library name in Python?